src/p_mobj.h

```diff
@@ -416,6 +416,9 @@ extern int iquetail;
 // [FG] colored blood and gibs
 extern boolean colored_blood;
 
+#define PLAYER_SLOPE(a)	((((a)->lookdir / MLOOKUNIT) << FRACBITS) / 173)
```

Comment:
I've been wondering: where does the `173` come from? On a related note, does this match the vertical aiming range of autoaim?

Reply:
Short answer, from Heretic: 😁 
https://github.com/chocolate-doom/chocolate-doom/blob/e96d2c2c1bf33b3367cf40d234563d7c5465b614/src/heretic/p_pspr.c#L818

Longer answer, honestly I don't know, but I'd love to de-obfuscate it.

Reply:
Maybe it's angle to fixed conversion? https://github.com/team-eternity/eternity/blob/cdac0335080c09e3eb38ed560de0de0f542cc3c7/source/tables.h#L106

Reply:
Also: https://github.com/team-eternity/eternity/blob/cdac0335080c09e3eb38ed560de0de0f542cc3c7/source/tables.h#L71-L73